crates/hir/src/symbols.rs

```diff
@@ -242,45 +242,84 @@ impl<'a> SymbolCollector<'a> {
         <<L as Lookup>::Data as HasSource>::Value: HasName,
     {
         self.push_file_symbol(|s| {
+            let mut symbols = Vec::new();
             let loc = id.lookup(s.db.upcast());
             let source = loc.source(s.db.upcast());
             let name_node = source.value.name()?;
-            Some(FileSymbol {
+            let def = ModuleDef::from(id.into());
+            let dec_loc = DeclarationLocation {
+                hir_file_id: source.file_id,
+                ptr: SyntaxNodePtr::new(source.value.syntax()),
+                name_ptr: SyntaxNodePtr::new(name_node.syntax()),
+            };
+
+            if let Some(attrs) = def.attrs(s.db) {
+                for alias in attrs.doc_aliases() {
+                    eprintln!("Adding alias {}", alias);
+                    symbols.push(FileSymbol {
+                        name: alias,
+                        def: def.clone(),
```

Comment:
nit: `ModuleDef` is `Copy` so you can avoid `clone()`. There's one more occurrence below.